ocs_ci/ocs/machine.py

```diff
@@ -120,3 +120,28 @@ def delete_machine_and_check_state_of_new_spinned_machine(machine_name):
         log.info(f"{new_machine.name} is in {state} state")
         return state == constants.STATUS_RUNNING.islower()
     return False
+
+
+def get_machinesets():
+    machines = list()
+    machinesets_obj = OCP(kind=constants.MACHINESETS, namespace=constants.OPENSHIFT_MACHINE_API_NAMESPACE)
+    for machine in machinesets_obj.get()['items']:
+        machines.append(machine.get('spec').get('selector').get(
+            'matchLabels').get('machine.openshift.io/cluster-api-machineset'))
+
+    return machines
```

Comment:
from function name, it looks like it returns machinests but the return variable name is 'machines'. make it consistent